transferui/src/main/java/com/hyperwallet/android/ui/transfer/CurrencyDetails.java

```diff
@@ -0,0 +1,132 @@
+package com.hyperwallet.android.ui.transfer;
+
+
+public class CurrencyDetails {
+
+    private Integer id;
+    private String name;
+    private String currencyCode;
+    private String isocurrencyCode;
+    private String baseunit;
+    private Integer denominationamount;
+    private Integer decimals;
+    private Integer hiddenDecimals;
+    private String symbol;
+    private Integer exchangeable;
+    private Integer governmentissued;
+    private Integer groupingUsed;
+    private Integer fxTransactionVisible;
+    private String displayedAs;
```

Comment:
follow java naming conventions, for ex.
`isocurrencyCode` should be `isoCurrencyCode`
`denominationamount` should be `denominationAmount`
etc..

Reply:
fixed